test/spec/todo/todo.spec.ts

```diff
@@ -120,3 +120,18 @@ describe("PUT /todos/:id", () => {
     }
   });
 });
+
+describe("GET /todos/:id", () => {
+  it("should return 200 if todo exists else 404", async () => {
```

Comment:
This is very poor test case.

A test case should only test one thing.

Reply:
done sir